api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml

```diff
@@ -0,0 +1,1202 @@
+                  <li>Microsoft.SignalRService/SignalR</li>
+                  <li>Microsoft.Sql/servers/databases</li>
+                  <li>Microsoft.Sql/servers/elasticPools</li>
+                  <li>Microsoft.Sql/managedInstances</li>
+                  <li>Microsoft.Storage/storageAccounts</li>
+                  <li>Microsoft.Storage/storageAccounts/blobServices</li>
+                  <li>Microsoft.Storage/storageAccounts/fileServices</li>
+                  <li>Microsoft.Storage/storageAccounts/queueServices</li>
+                  <li>Microsoft.Storage/storageAccounts/tableServices</li>
+                  <li>Microsoft.StreamAnalytics/streamingjobs</li>
+                  <li>Microsoft.TimeSeriesInsights/environments</li>
+                  <li>Microsoft.TimeSeriesInsights/environments/eventsources</li>
+                  <li>Microsoft.Web/serverfarms</li>
+                  <li>Microsoft.Web/sites (excluding functions)</li>
+                  <li>Microsoft.Web/sites (functions)</li>
+                  <li>Microsoft.Web/sites/slots</li>
+                  <li>Microsoft.Web/hostingEnvironments/multiRolePools</li>
+                  <li>Microsoft.Web/hostingEnvironments/workerPools</li>
+                  </ul>"
+                type: string
+                example: "Microsoft.Network/applicationGateways"
+                default: Microsoft.Compute
+              operator:
+                title: Operator
+                description: "The operator used to evaluate the threshold against the current metric value.
+                   <ul>
+                   <li>gt (Greater Than)</li>
+                   <li>gte (Greater Than or Equal)</li>
+                   <li>lt (Less Than)</li>
+                   <li>lte (Less Than or Equal)</li>
+                   <ul>"
+                type: string
+                example: "gte"
+                enum:
+                  - gt
+                  - gte
+                  - lt
+                  - lte
+              dimenstions:
+                title: Dimenstions
+                description: TBD
+                type: array
+                items:
+                  type: object
+                  properties:
+                    name:
+                      title: TBD
+                      description: TBD
+                      type: string
+                      example: resourceName
+                    value:
+                      title: TBD
+                      description: TBD
+                      type: string
+                      example: TestAppGateway
+              action:
+                title: Action
+                description: Scaling action to take when the policy is triggered.
+                type: object
+                properties:
+                  type:
+                    title: Type
+                    description: "Type of scaling action to take when the scaling policy is triggered. </br> Valid Values:
+                    <ul>
```

Comment:
@vlad666spot you are using HTML syntax here, while we use Markdown.
please adjust accordingly in the whole schema (you have an example on Azure V3)

Reply:
![Screenshot 2020-11-03 at 13 26 27](https://user-images.githubusercontent.com/64230439/97979684-44d6c780-1dd8-11eb-8657-c0b8a13c22b4.png)
Asked several times to Yarden, but didn't got answer, so decided to put link on my own (https://docs.microsoft.com/en-us/azure/templates/ )

Reply:
Fixed